docs/source/user_guide/runtime-conf.md

```diff
@@ -115,6 +116,7 @@ elyra-metadata install runtimes \
        --name="my_kubeflow_pipelines_runtime" \
        --display_name="My Kubeflow Pipelines Runtime" \
        --api_endpoint=https://kubernetes-service.ibm.com/pipeline \
+       --auth_type="DEX with static password" \
```

Comment:
Is `--auth_type="DEX with static password"` actually how you refer to `DEX_STATIC_PASSWORDS`, that seems a bit crazy.

Why not `--auth_type=DEX_STATIC_PASSWORDS`?